bin/ingest_aclpub2.py

```diff
@@ -0,0 +1,581 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright 2021, 2022 Xinru Yan <xinru1414@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Usage:
+#   python bin/ingest_aclpub2.py
+#
+#
+#
+from collections import defaultdict
+import click
+import yaml
+import re
+import sys
+import os
+import glob
+from pathlib import Path
+from datetime import datetime
+from collections import defaultdict
+from itertools import chain
+import lxml.etree as etree
+from typing import Dict, List, Tuple, Any, Optional
+from ingest import maybe_copy
+
+from normalize_anth import normalize
+from anthology.index import AnthologyIndex
+from anthology.venues import VenueIndex
+from anthology.people import PersonName
+from anthology.utils import (
+    make_simple_element,
+    indent,
+    compute_hash_from_file,
+)
+
+
+def disambiguate_name(node, anth_id, people):
+    name = PersonName.from_element(node)
+    ids = people.get_ids(name)
+    choice = -1
+    if len(ids) > 1:
+        while choice < 0 or choice >= len(ids):
+            print(
+                f'({anth_id}): ambiguous author {name}; Please choose from the following:'
+            )
+            for i, id_ in enumerate(ids):
+                print(f'[{i}] {id_} ({people.get_comment(id_)})')
+            choice = int(input("--> "))
+
+    return ids[choice], choice
+
```

Comment:
Best to import these from `ingest.py`, just to reduce redundancy. (You probably have to move them first, since they're inside `main()`)